.github/workflows/publish-helm-charts.yml

```diff
@@ -89,3 +105,21 @@ jobs:
           add: '.'
           cwd: './airbyte-oss/'
       
+      - name: Commit and push changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: 'Bump release to ${{ needs.generate-semantic-version.outputs.tag }}'
+          add: '.'
```

Comment:
do we need to set current working directory to defend against changes to this script that may move current working directory?  

I'm not sure if new actions all start in the same spot or not